website/content/docs/release-notes/consul-terraform-sync/v0_6_x.mdx

```diff
@@ -0,0 +1,46 @@
+---
+layout: docs
+page_title: 0.4.x
+description: >-
+  Consul-Terraform-Sync release notes for version 0.6.x
+---
+
+# Consul-Terraform-Sync 0.6.0
+
+## Release Highlights
+
+Per-Task Execution Mode with Support for Terraform Cloud Agent (Enterprise): CTS now allows users to select the Terraform execution mode to be “remote” or “cloud agent” per Task. For more information refer to the [per-task execution mode documentation](/docs/nia/network-drivers/terraform-cloud#remote-workspaces).
+
+HCP Consul support (Enterprise): CTS now supports interoperability with HCP Consul with license retrieval capabilities that allow users to keep licenses in sync with their HCP Consul or Consul enterprise deployment.
+
+Auto Service-Registration with Consul, and Health API Endpoint: CTS now supports a new configuration block that auto registers CTS as a service within Consul as soon as the CTS is instantiated. This eliminates the manual step of registering CTS as a service within the Consul catalog. This service registration also enables health checking on CTS using the newly added health API endpoint.
```

Comment:
```suggestion
Auto Service-Registration with Consul, and Health API Endpoint: CTS now supports a new configuration block that auto registers CTS as a service within Consul as soon as CTS is instantiated. This eliminates the manual step of registering CTS as a service within the Consul catalog. This service registration also enables health checking on CTS using the newly added health API endpoint.
```

Reply:
Should we add a link to the health API endpoint docs?

Reply:
Good idea, I updated your suggestion to include the health endpoint url